packages/rest/src/client/doc/IRestClientResponse.ts

```diff
@@ -0,0 +1,33 @@
+
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { Writable, Readable } from "stream";
+import { ITaskWithStatus } from "../../../../operations";
+import { Session } from "../../session/Session";
+import { Logger } from "../../../../logger";
+
+export interface IRestClientResponse {
+    requestSuccess?: boolean;
+    requestFailure?: boolean;
+    data?: Buffer;
+    dataString?: string;
+    response?: any;
+    session?: Session;
+    log?: Logger;
+}
+
+// requestFailure: "requestFailure" as CLIENT_PROPERTY,
```

Comment:
perhaps these old comments can be deleted

Reply:
Ah, I thought I got them all. Thanks for catching this.

Reply:
done with a2f090a